tests/unit/participants/test_views.py

```diff
@@ -640,6 +646,8 @@ def test_when_there_is_no_participant_team_of_user(self):
         }
 
         response = self.client.get(self.url, {})
+        self.assertTrue(abs(response.data['datetime_now'] - self.time) < timedelta(seconds=1))
+        del response.data['datetime_now']
```

Comment:
Can you please add a comment that why are we deleting this here (add the same comment everywhere wherever you are deleting it)? it would be easier for someone who will be visiting this code for the first time.

Reply:
@deshraj Sure